### ISSUE TITLE
Add ability to pause animated background

### ISSUE THREAD

**Issue by dhershman1** [dhershman1/dusty-codes-site#2]
The background animation currently does not have an ability to pause it which may be an accessibility issue.

I want to give the ability to pause it

**Comment by dhershman1** [comment]
The animated background has been removed in favor of a still professional background
